### PR TITLE
feat(#16): :sparkles: Provide your own environment variables for uptrace container

### LINKED ISSUE #16: Unable to set external postgress credentials via secrets
As title said: helm expects from me `user` and `password` in plain text. I want to set the values via secrets

### PR DESCRIPTION
**Pull request by numfin**
This PR gives ability to set your own environment variables for uptrace container
structure is compatible with https://kubernetes.io/docs/tasks/inject-data-application/define-environment-variable-container/

Closes #16

### CODE REVIEW

File: charts/uptrace/values.yaml [hunk 1]
```diff
@@ -108,6 +108,11 @@ tolerations: []
 
 affinity: {}
 
+## Provided env variables for uptrace container
+## You can use those variables in uptrace.config
+## Values are same as here https://kubernetes.io/docs/tasks/inject-data-application/define-environment-variable-container/
+providedEnv: {}
```
Comment (vmihailenco) [inline charts/uptrace/values.yaml:114]: What do you think about moving this to `uptrace` section so we have `uptrace.env` and `uptrace.config`?

Comment (numfin) [reply charts/uptrace/values.yaml:114]: sure

**Comment by vmihailenco** [comment]
Thanks. TBH I don't see why we need to define env variables in `uptrace.env` instead of changing `uptrace.config` directly, but I don't mind having this.

**Comment by numfin** [comment]
@vmihailenco for example if we need to provide Postgres password - we ask uptrace to use value from secrets, instead of deploying another helm release. If we have secret manager like "doppler" it's really usefull. Also i don't like to share secrets in my org repository :D

**Pull request merged**